data/_nav.yml

```diff
@@ -384,8 +384,12 @@ tutorials:
       content:
         - url: /docs/tutorials/native/intro-to-kotlin-native.html
           title: Introduction to Kotlin/Native
```

Comment:
The Hands-on under this link seems more advanced than the next three topics. Should we rename the link to reflect content (HTTP client) and move in lower in the list?

Reply:
Yes. Agree. Will do.